integration-cli/docker_cli_exec_test.go

```diff
@@ -509,16 +509,18 @@ func (s *DockerCLIExecSuite) TestExecUlimits(c *testing.T) {
 
 // #15750
 func (s *DockerCLIExecSuite) TestExecStartFails(c *testing.T) {
-	// TODO Windows CI. This test should be portable. Figure out why it fails
-	// currently.
-	testRequires(c, DaemonIsLinux)
 	name := "exec-15750"
 	runSleepingContainer(c, "-d", "--name", name)
 	assert.NilError(c, waitRun(name))
 
 	out, _, err := dockerCmdWithError("exec", name, "no-such-cmd")
 	assert.ErrorContains(c, err, "", out)
-	assert.Assert(c, strings.Contains(out, "executable file not found"))
+
+	expectedMsg := "executable file not found"
+	if DaemonIsWindows() {
+		expectedMsg = "The system cannot find the file specified"
+	}
```

Comment:
Still trying to figure out if there's a canonical definition of these errors. I know we have other quirky things like https://github.com/moby/moby/blob/1855a55d8c3cba353e6007f0ab355004ebe478d4/daemon/errors.go#L182-L189


I did find the Linux error in the `os/exec` package; https://github.com/golang/go/blob/4a3071696ddfb13e1a8f35f76197b7b3143492f4/src/os/exec/lp_unix.go#L19-L20

But couldn't find something like that for Windows. Did find a reference here;
https://github.com/microsoft/terminal/blob/7a2e4f8d9b521e3f0d645510841a918150b64a25/oss/fmt/include/fmt/os.h#L182-L199

But I expect the actual error to be coming from Windows itself?

Reply:
The actual error is the error code [`ERROR_FILE_NOT_FOUND`](https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-#:~:text=ERROR_FILE_NOT_FOUND) stringified [using `FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM, ...)`](https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-#:~:text=To%20retrieve%20the%20description%20text%20for%20the%20error%20in%20your%20application%2C%20use%20the%20FormatMessage%20function%20with%20the%20FORMAT_MESSAGE_FROM_SYSTEM%20flag.) Like its `strerror(3)` equivalent on POSIX, the error string might be localized.